src/src/rx_main.cpp

```diff
@@ -1058,6 +1058,11 @@ static bool ICACHE_RAM_ATTR ProcessRfPacket_SYNC(uint32_t const now, OTA_Sync_s
     DBGW('s');
 #endif
 
+    if (isDualRadio())
+    {
+        config.SetAntennaMode(otaSync->geminiMode);
```

Comment:
If the antenna mode is not configurable on the RX then we should remove it from the config!

Reply:
This might add complexity in managing the config.  MCUs like the 8285 are used on single and diversity receivers.  So we dont gain anything in saving eeprom.

Reply:
Ah, yeah, right it's shared with Diversity receivers.